OracleAccessManagement/dockerfiles/12.2.1.4.0/Dockerfile

```diff
@@ -0,0 +1,176 @@
+# ----------------------------------------------------------------
+USER root
+RUN mkdir -p ${BASE_DIR} && \
+#    useradd -b ${BASE_DIR} -d ${ORACLE_HOME} -m -s /bin/bash oracle && \
+    chmod a+xr ${BASE_DIR} && chown oracle:oracle ${BASE_DIR} && \
+    mkdir -p ${USER_PROJECTS_DIR} && \
+    chown -R oracle:oracle ${USER_PROJECTS_DIR} && chmod -R 775 ${USER_PROJECTS_DIR} && \
+    mkdir -p ${CONTAINER_DIR} && \
+    chown -R oracle:oracle ${CONTAINER_DIR} && chmod -R 775 ${CONTAINER_DIR} && \
+    mkdir -p ${SCRIPT_DIR} && chown oracle:oracle ${SCRIPT_DIR} && \
+    mkdir -p ${PROPS_DIR} && chown oracle:oracle ${PROPS_DIR} && \ 
+    yum install -y libaio && \
+    yum install -y hostname && \
+    rm -rf /var/cache/yum && \
+    mkdir ${PATCH_DIR} && \
+    mkdir ${OPATCH_PATCH_DIR} && \
+    chown -R oracle:oracle ${BASE_DIR} && \
+    chown -R oracle:oracle ${PATCH_DIR} && \
+    chown -R oracle:oracle ${OPATCH_PATCH_DIR}
+
+#
+FROM base as builder
+# Copy packages and scripts
+# -------------------------
+COPY --chown=oracle:oracle Dockerfile patches/* ${PATCH_DIR}/
+COPY --chown=oracle:oracle Dockerfile opatch_patch/* ${OPATCH_PATCH_DIR}/  
+COPY container-scripts/* ${SCRIPT_DIR}/
+COPY install/* ${BASE_DIR}/
+ADD  $FMW_IDM_JAR ${BASE_DIR}/
+
+#
+# Update Permissions for packages and scripts
+# --------------------------------------------
+RUN cd ${BASE_DIR} && chmod 755 *.jar && \
+     chmod a+xr ${SCRIPT_DIR}/* && \
+     chown -R oracle:oracle ${CONTAINER_DIR} && chmod -R 775 ${CONTAINER_DIR} && \
+     chown oracle:oracle ${SCRIPT_DIR}/*
+
+# 
+# Installation of IDM Binaries
+# --------------------------------------------
+USER oracle
+WORKDIR ${ORACLE_HOME}
+RUN cd ${BASE_DIR} && \
+#install IDM in silent mode
+ $JAVA_HOME/bin/java -jar ${BASE_DIR}/$FMW_IDM_JAR -silent -responseFile ${BASE_DIR}/iam.response -invPtrLoc ${ORACLE_HOME}/oraInst.loc -jreLoc $JAVA_HOME -ignoreSysPrereqs -force -novalidation ORACLE_HOME=${ORACLE_HOME} && \
+ rm -fr ${BASE_DIR}/*.jar ${BASE_DIR}/*.response && \
+ rm -f ${OPATCH_PATCH_DIR}/Dockerfile && \
+ rm -f ${PATCH_DIR}/Dockerfile
+
+#
+# Apply patch to OPatch
+#
+USER oracle
+WORKDIR ${ORACLE_HOME} 
+RUN opatchzip=`ls ${OPATCH_PATCH_DIR}/p*.zip 2>/dev/null`; \
+    if [ ! -z "$opatchzip" ]; then \
+      cd ${OPATCH_PATCH_DIR};  \
+      echo -e "\nBelow patch present in opatch_patch directory. Applying this patch:" ; \
+      ls p*.zip ; \
+      echo -e "" ; \
+      opatchfile=`ls p*.zip` ; \
+      $JAVA_HOME/bin/jar xf $opatchfile ; \
+      $JAVA_HOME/bin/java -jar ${OPATCH_PATCH_DIR}/6880880/opatch_generic.jar -silent oracle_home=$ORACLE_HOME; \
+      if [ $? -ne 0 ]; then \
+        echo "Applying patch to opatch Failed" ; \
+        exit 1 ; \
+      fi; \      
+      cd /tmp; \
+      rm  ${OPATCH_PATCH_DIR}/*.zip; \
+      rm -r ${OPATCH_PATCH_DIR}/; \
+    fi  
+#
+# Apply product patches 
+#    
+USER oracle
+WORKDIR ${ORACLE_HOME}
```

Comment:
The `WORKDIR` is already `${ORACLE_HOME}` (set on line 101). This directive isn't needed.

Reply:
Will take care of it.